### ISSUE TITLE
Flyte Charm function is backwards

### ISSUE THREAD

**Issue by Dayyer1** [yorkeJohn/Tainted-Magic#51]
You boost forward when you're falling and press shift, instead of sprinting while flying to boost forward.

**Comment by Dayyer1** [comment]
Also, I just tried to use the Lumos focus, but the upgrade ID's seem to be messed up. For example, the Glowpet was replaced with Sanity.

**Comment by Dayyer1** [comment]
(Special message: due to certain circumstances [during the past two years] , I have been unable to learn any programming/java/etc., but I am now free and able to learn. Hopefully I can help in the future instead of just reporting bugs to you.)

**Comment by yorkeJohn** [comment]
I can assure you that the flyte charm works as intended. When falling you do glide forwards and down and when flying you press the sprint key to quickly burst forwards. I might lower the multiplier so you don't go forwards as fast when gliding. 

As for the foci, I changed the upgeade ids because I got a weird crash (which probably wasn't tainted magic related anyways) and didn't realize it would mess up people's foci, I will change them back.
